pine64/rockpro64/default.nix

```diff
@@ -0,0 +1,9 @@
+{ pkgs, lib, ... }:
```

Comment:
Please reference this in the flake.nix and in README.md so that it's tested and discoverable.